specifications/10-flexible-rollout-strategy.json

```
@@ -0,0 +1,142 @@
+                            "stickiness": "userId",
+                            "groupId": "Feature.flexibleRollout.userId.55"
+                        },
+                        "constraints": []
+                    }
+                ]
+            },
+            {
+                "name": "Feature.flexibleRollout.sessionId.42",
+                "description": "Should be enabled",
+                "enabled": true,
+                "strategies": [
+                    {
+                        "name": "flexibleRollout",
+                        "parameters": {
+                            "rollout": "42",
+                            "stickiness": "sessionId",
+                            "groupId": "Feature.flexibleRollout.sessionId.42"
+                        },
+                        "constraints": []
+                    }
+                ]
+            }
+        ]
+    },
+    "tests": [
+        {
+          "description": "Feature.flexibleRollout.100 should always be enabled",
+          "context": {},
+          "toggleName": "Feature.flexibleRollout.100",
+          "expectedResult": true
+        },
+        {
+            "description": "Feature.flexibleRollout.10 should be enabled for userId=174",
+            "context": {
+                "userId": "174"
+            },
+            "toggleName": "Feature.flexibleRollout.10",
+            "expectedResult": true
+        },
+        {
+            "description": "Feature.flexibleRollout.10 should be disabled for userId=499",
+            "context": {
+                "userId": "499"
+            },
+            "toggleName": "Feature.flexibleRollout.10",
+            "expectedResult": false
+        },
+        {
+            "description": "Feature.flexibleRollout.10 should be enabled for sessionId=174",
+            "context": {
+                "sessionId": "174"
+            },
+            "toggleName": "Feature.flexibleRollout.10",
+            "expectedResult": true
+        },
+        {
+            "description": "Feature.flexibleRollout.10 should be disabled for sessionId=499",
+            "context": {
+                "sessionId": "499"
+            },
+            "toggleName": "Feature.flexibleRollout.10",
+            "expectedResult": false
+        },
+        {
+            "description": "Feature.flexibleRollout.userId.55 should be enabled for userId=25",
+            "context": {
+                "userId": "25"
+            },
+            "toggleName": "Feature.flexibleRollout.userId.55",
+            "expectedResult": true
+        },
+        {
+            "description": "Feature.flexibleRollout.userId.55 should be disabled for sessionId=25",
+            "context": {
+                "sessionId": "25"
+            },
+            "toggleName": "Feature.flexibleRollout.userId.55",
+            "expectedResult": false
+        },
+        {
+            "description": "Feature.flexibleRollout.userId.55 should be disabled",
```

```suggestion
            "description": "Feature.flexibleRollout.userId.55 should be disabled if no userId in context",
```
Just documentation cleanup. :)